src/lib.rs

```diff
@@ -40,7 +40,8 @@ impl<R: gfx::Resources> Texture<R> {
     pub fn empty<F>(factory: &mut F) -> Result<Self, CombinedError>
         where F: gfx::Factory<R>
     {
-        Rgba8Texture::create(factory, &[0u8; 4], [1, 1], &TextureSettings::new())
+        CreateTexture::create(factory, Format::Rgba8, &[0u8; 4], [1, 1],
```

Comment:
If Piston defaults to `Srgba8` for the render target, and to `Rgba8` for textures, any attempt to straight 1:1 display a texture will look washed out, beware.